Resources/Prototypes/Announcers/!randomAnnouncers.yml

```diff
@@ -2,9 +2,10 @@
   id: RandomAnnouncers
   weights:
     Intern: 0.1
-    MedBot: 0.6
-    Michael: 0.3
+    MedBot: 0.4
+    Michael: 0.2
     NEIL: 0.3
     TikTok: 0.1
     VoxFem: 1.2
-    Stern: 0.2
+    Stern: 0.25
```

Comment:
did you intend to make Stern more common? not documented on your changelog

Reply:
Oh shit yeah; I was talking with Kazne about making the custom ones equal in weight, kinda like a middle between the common announcers and the sillier ones like TikTok and Intern

Reply:
okay cool just checking. i only remembered talking about making michael and medbot less common and this could have plausibly been a typo

Reply:
Updated to reflect; sorry if any of this is super scuffed. My first ever PR and first time ever using GitHub